handler/preprocess.sh

```diff
@@ -35,6 +36,48 @@ echo "finding dicom directories"
 ./find_dicomdir.py $root > $root/dcm2niix.list
 cat $root/dcm2niix.list
```

Comment:
I'd say to use `mktemp` to create temporary files and make sure they are removed after the script finishes. It helps with cluttering and possible misbehaviors.

Reply:
Well, all I did was add the echo, but I agree with you. We should only need to use temp folders (for pet2bids) for the `*.list`? files, pet2bids already does all its work in temporary directories. I should probably sync this with the latest from master to see if that's been implemented.

Reply:
Yes, it was my first time looking through this script tbh, wasnt aware that you were replicating what's in the code already until later in the game.

using `mktemp` instead of `$root/dcm2niix.list` would create an unused path, that you can use as a file or folder, so I think this case would be a file.

maybe it is fine the way it is now, for debugging, what you think?